frontend/web/components/Aside.js

```diff
@@ -91,8 +91,8 @@ const Aside = class extends Component {
 
   render() {
     const { asideIsVisible, disabled, toggleAside } = this.props
-    let integrations = Utils.getFlagsmithValue('integrations') || '[]'
-    integrations = JSON.parse(integrations)
+    let integrations = Utils.getFlagsmithValue('integration_data') || '{}'
+    integrations = Object.keys(JSON.parse(integrations))
```

Comment:
I don't love the reassignment here, I think we should have 2 separate variables here, or just combine this into a single variable assignment? 

(Same comment on the `IntegrationsPage` diff too.

Reply:
Ok I think we can do the latter since the FE waits for Flagsmith to have values